### ISSUE TITLE
Error with new undetected_chromedriver

### ISSUE THREAD

**Issue by marcodallagatta** [flathunters/flathunter#310]
Since the last couple of days, I've had this problem. Tried to update Chrome and deleting the undetected_chromedriver folder without success:

```
C:\Users\User\Desktop\flathunter-main>pipenv run python flathunt.py
[2023/02/10 07:59:10|config.py               |INFO    ]: Using config path C:\Users\User\Desktop\flathunter-main/config.yaml
[2023/02/10 07:59:10|logging.py              |DEBUG   ]: Settings from config: { ... }
[2023/02/10 07:59:10|chrome_wrapper.py       |INFO    ]: Initializing Chrome WebDriver for crawler...
[2023/02/10 07:59:12|patcher.py              |INFO    ]: patching driver executable C:\Users\User\appdata\roaming\undetected_chromedriver\undetected_chromedriver.exe
Traceback (most recent call last):
  File "C:\Users\User\Desktop\flathunter-main\flathunt.py", line 109, in <module>
    main()
  File "C:\Users\User\Desktop\flathunter-main\flathunt.py", line 75, in main
    config.init_searchers()
  File "C:\Users\User\Desktop\flathunter-main\flathunter\config.py", line 98, in init_searchers
    CrawlImmobilienscout(self),
    ^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\User\Desktop\flathunter-main\flathunter\crawl_immobilienscout.py", line 40, in __init__
    self.driver = get_chrome_driver(driver_arguments)
                  ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\User\Desktop\flathunter-main\flathunter\chrome_wrapper.py", line 44, in get_chrome_driver
    driver = uc.Chrome(version_main=chrome_version, options=chrome_options) # pylint: disable=no-member
             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\User\.virtualenvs\flathunter-main-tiVfcs-e\Lib\site-packages\undetected_chromedriver\__init__.py", line 431, in __init__
    super(Chrome, self).__init__(
  File "C:\Users\User\.virtualenvs\flathunter-main-tiVfcs-e\Lib\site-packages\selenium\webdriver\chrome\webdriver.py", line 80, in __init__
    super().__init__(
  File "C:\Users\User\.virtualenvs\flathunter-main-tiVfcs-e\Lib\site-packages\selenium\webdriver\chromium\webdriver.py", line 104, in __init__
    super().__init__(
  File "C:\Users\User\.virtualenvs\flathunter-main-tiVfcs-e\Lib\site-packages\selenium\webdriver\remote\webdriver.py", line 286, in __init__
    self.start_session(capabilities, browser_profile)
  File "C:\Users\User\.virtualenvs\flathunter-main-tiVfcs-e\Lib\site-packages\undetected_chromedriver\__init__.py", line 694, in start_session
    super(selenium.webdriver.chrome.webdriver.WebDriver, self).start_session(
  File "C:\Users\User\.virtualenvs\flathunter-main-tiVfcs-e\Lib\site-packages\selenium\webdriver\remote\webdriver.py", line 378, in start_session
    response = self.execute(Command.NEW_SESSION, parameters)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\User\.virtualenvs\flathunter-main-tiVfcs-e\Lib\site-packages\selenium\webdriver\remote\webdriver.py", line 440, in execute
    self.error_handler.check_response(response)
  File "C:\Users\User\.virtualenvs\flathunter-main-tiVfcs-e\Lib\site-packages\selenium\webdriver\remote\errorhandler.py", line 245, in check_response
    raise exception_class(message, screen, stacktrace)
selenium.common.exceptions.WebDriverException: Message: unknown error: cannot connect to chrome at 127.0.0.1:14432
from unknown error: unable to discover open pages
Stacktrace:
Backtrace:
        (No symbol) [0x011037D3]
        (No symbol) [0x01098B81]
        (No symbol) [0x00F9B36D]
        (No symbol) [0x00FBED6D]
        (No symbol) [0x00FB70F0]
        (No symbol) [0x00FB6EE1]
        (No symbol) [0x00FF1ED5]
        (No symbol) [0x00FF1B2C]
        (No symbol) [0x00FEB216]
        (No symbol) [0x00FC0D97]
        (No symbol) [0x00FC253D]
        GetHandleVerifier [0x0137ABF2+2510930]
        GetHandleVerifier [0x013A8EC1+2700065]
        GetHandleVerifier [0x013AC86C+2714828]
        GetHandleVerifier [0x011B3480+645344]
        (No symbol) [0x010A0FD2]
        (No symbol) [0x010A6C68]
        (No symbol) [0x010A6D4B]
        (No symbol) [0x010B0D6B]
        BaseThreadInitThunk [0x77247D69+25]
        RtlInitializeExceptionChain [0x77A5BB9B+107]
        RtlClearBits [0x77A5BB1F+191]

[2023/02/10 08:00:14|__init__.py             |INFO    ]: ensuring close
Exception ignored in: <function Chrome.__del__ at 0x0000020CEEC231A0>
Traceback (most recent call last):
  File "C:\Users\User\.virtualenvs\flathunter-main-tiVfcs-e\Lib\site-packages\undetected_chromedriver\__init__.py", line 778, in __del__
  File "C:\Users\User\.virtualenvs\flathunter-main-tiVfcs-e\Lib\site-packages\undetected_chromedriver\__init__.py", line 733, in quit
OSError: [WinError 6] The handle is invalid

[process exited with code 1 (0x00000001)]
```

**Comment by codders** [comment]
Thanks for the report. I can only test on linux, but what I see here is:

```
    raise exception_class(message, screen, stacktrace)
selenium.common.exceptions.WebDriverException: Message: unknown error: Runtime.evaluate threw exception: SyntaxError: missing ) after argument list
  (Session info: chrome=109.0.5414.119)
```

It looks like there are some issues with the new undetected-chromedriver version: https://github.com/ultrafunkamsterdam/undetected-chromedriver/issues/1035 . I guess we have to wait and see what kind of fixes show up from there. In the meantime, rolling back to flathunter@4d94465 should rollback the latest driver upgrade.

**Comment by codders** [comment]
Just merged in #313, which bumps `undetected-chromedriver` up a version. Maybe try again and see if that's better?

**Comment by friedrichtroescher** [comment]
Unfortunately, the problem remains unchanged after updating. Also, i cant seem to find a version with ID 4d94465 - could you send me a link to a page for it? Or could it be a typo?

**Comment by marcodallagatta** [comment]
> Just merged in #313, which bumps `undetected-chromedriver` up a version. Maybe try again and see if that's better?

Working with latest!

**Comment by codders** [comment]
@friedrichtroescher With 4d94465 I meant that you could try:

```
git checkout 4d94465
```

That should work, but you won't be on any tag or branch - you can make a branch for yourself with `git branch`. But it looks like the fix worked for @marcodallagatta
